configure

```diff
@@ -276,41 +278,52 @@ elif (args.eigen != ""):
     f.write("EIGEN = " + args.eigen + "\n")
 
 #
-# Compiler
+# Compiler and MPI library
 #
 if   (args.comp == "g++"):     f.write("COMP = GCC\n")
 elif (args.comp == "clang++"): f.write("COMP = CLANG\n")
 elif (args.comp == "icc"):     f.write("COMP = INTEL\n")
 else: raise Exception(color.red + "ERROR" + color.reset + ": Compiler must be g++, clang++, or icc")
 
-message("Comp-Type", str(args.comp))
-shellmessage("Comp-Version",args.comp+" --version")
-shellmessage("Comp-MPI","mpicxx -cxx="+args.comp+" -show")
-f.write("CC = mpicxx -cxx="+args.comp+"\n")
+if not args.alternate_mpi and not shutil.which("mpichversion"):
```

Comment:
@bsrunnels, your last commit breaks the default behaviour, this should read `if not args.alternate_mpi:`.

Reply:
@nmnobre Fixed, thank you